src/common-models/worker-info.model.ts

```diff
@@ -23,3 +23,9 @@ export interface WorkerInfoModel {
   name: string;
   time: number;
 }
+
+export interface WorkerData {
```

Comment:
Utworzyłem kolejny interfejs do danych o pracowniku. W przyszłości trzeba to będzie zrefactorować i zostawić jeden XD

Reply:
Ja bym rozszerzył ten co już istnieje dodając do niego opcjonalne propercje

Reply:
Kurde faktycznie, nie pomyślałem o tym. Poprawię :D